### ISSUE TITLE
Question about dataset.

### ISSUE THREAD

**Issue by zhenshan0127** [Lan1eve/SVDC#4]
Hello,

Thanks for your amazing work and appreciate for the code. I notice that you utilized dToF sparse depth maps that were sampled and processed from ground truth depth as input, and the ground truth depth is reliable because it comes from simulation. So, I have two questions:

(1) Is this "raw depth" same as or similar to the actual incomplete and noisy raw depth directly obtained from the depth sensor? 
(2) If I want to use the actual incomplete and noisy raw depth and the corresponding grount truth depth, could you please give me some ideas to find it?

Thanks for your time and patience.

Regards,
Zhenshan

**Comment by Lan1eve** [comment]
(1) Our dToF sparse depth data is generated by simulating an actual dToF sensor, resulting in incomplete and noisy characteristics. The reason for this approach is that it is challenging to obtain a large amount of high-quality real dToF data and corresponding depth ground truth. Therefore, we conducted simulations, trained and tested our model on synthetic datasets, and finally tested its performance using real dToF sensor data.

(2) You can simulate synthetic data by modeling the noise patterns or missing pixel distributions present in actual raw depth data. Alternatively, you can collect real-world data using actual dToF sensors.

**Comment by zhenshan0127** [comment]
Hello @Lan1eve,

Thanks for your reply and explanation. Does the real dToF sensor data you used finally refer to the TetrasRGBD dataset mentioned in paper, MIPI 2023 Challenge on RGB+ToF Depth Completion: Methods and Results? Because I find that other datasets (TartanAir, DynamicReplica, DydToF) are synthetic. Please correct me if I'm wrong.

Thanks for your time and patience.

Regards,
Zhensahn

**Comment by Lan1eve** [comment]
I noticed that the paper at [MIPI 2022 Challenge on RGB+ToF Depth Completion: Dataset and Report](https://arxiv.org/abs/2209.07057) states the Tetra-S RGBD training set consists of synthetic data, while the test set is from mixed sources. I'm unsure if the test set includes real dToF sensor data, and I also don't know if it will remain available for download after the competition concludes.
However, we used a real dToF sensor from the company and conducted tests with datasets captured by the company.

**Comment by zhenshan0127** [comment]
Hello @Lan1eve ,

I appreciate for your clarification.  So, you did not put the test results from company dataset in the paper, right? And it's impossible to share the dataset captured by company due to business reasons.

Thanks for your time and patience.

Regards,
Zhenshan

**Comment by Lan1eve** [comment]
Thank you for your attention. Yes, that's exactly what you said.

**Comment by zhenshan0127** [comment]
Thank you very much!
